containers/elm-0.19.0/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+FROM debian:9
+
+# Avoid warnings by switching to noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Configure apt and install packages
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils 2>&1 \ 
+    #
+    # Verify git and needed tools are installed
+    && apt-get install -y git procps \
+    && apt-get install -y wget \
+    #
+    # Install elm globally
+    && wget -O - https://github.com/elm/compiler/releases/download/0.19.0/binary-for-linux-64-bit.gz | gunzip -c > /usr/local/bin/elm && chmod +x /usr/local/bin/elm \
```

Comment:
It might be worth moving the elm version info an `ARG`

e.g. 

`ARG ELM_VERSION=0.19.0`

then later

` wget -O - https://github.com/elm/compiler/releases/download/${ELM_VERSION}/binary-for-linux-64-bit.gz | gunzip -c > /usr/local/bin/elm && chmod +x /usr/local/bin/elm`

Reply:
Good call! 😊
Resolved in [b1978a](https://github.com/microsoft/vscode-dev-containers/pull/101/commits/b1978a32683a28db1e76d346c2841607b50cf959)